lib/assets/javascripts/views/plans/new.js

```diff
@@ -38,29 +40,27 @@ $(() => {
   const getMethod = jQueryForm => jQueryForm.attr('method');
 
   // When one of the autocomplete fields changes, fetch the available templates
-  const handleComboboxChange = () => {
-    if ($('#plan_org_id').length && $('#plan_funder_id').length) {
-      const validOrg = ($('#plan_org_id').val().trim().length > 0 || $('#plan_no_org').prop('checked'));
-      const validFunder = ($('#plan_funder_id').val().trim().length > 0 || $('#plan_no_funder').prop('checked'));
+  const handleComboboxChange = debounce(() => {
+    const validOrg = (isValidText($('#plan_org_id').val()) || $('#plan_no_org').prop('checked'));
```

Comment:
isValidNumber should equally work since tries to convert the string to a number. Moreover, as the id are integers you would ensure that a valid id is sent back to the server.